projects/cashmere-examples/src/lib/modal-overview/modal-overview-example.component.ts

```diff
@@ -18,7 +18,7 @@ export class ModalOverviewExampleComponent {
         const options: ModalOptions = {
             data: {
                 content: 'This is a size lg modal. If isDraggable is set to true, it places a drag handle in the top right corner which you can use to reposition the modal. If isResizable is set to true, a resize handle appears in the bottom right to change the dimensions of the modal. They may be used together or separately.',
-                tight: false
```

Comment:
This is a case of confusing documentation.  Here we are passing an object to the example modal's component, and we're trying to tell it whether it's a tight modal or not to display appropriate content.  But we used a boolean flag with the same name (`tight`) as the actual setting - so it looks very confusing to a developer whether you set tight in the modal's data outside of it.  So that's why I just changed the object key name to be less misleading.